src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx

```diff
@@ -380,13 +385,14 @@ export const ArtworkSidebarCommercialButtons: React.FC<ArtworkSidebarCommercialB
         </>
       )}
 
-      {shouldRenderButtons && <Spacer y={2} />}
+      {Object.keys(renderButtons).length > 0 && <Spacer y={2} />}
 
       <Flex flexDirection={["column", "column", "column", "column", "row"]}>
         <Join separator={<Spacer x={1} y={1} />}>
           <AlertSwitch />
-          {buyNowOrPartnerOfferAvailable && (
+          {renderButtons["buyNow"] && (
```

Comment:
`renderButtons.buyNow`

Reply:
To make it work this way seems like I need to be more strict with object typing. But maybe it's a good thing. What you think now @damassi ?

Reply:
Yah we should type stuff! 

```
const foo: {
  buyNow: boolean
  makeOffer: boolean
} = {
 ... 
}